library_management_system_downloader/evaluate_api_response_for_fulltext_indication.py

```diff
@@ -0,0 +1,40 @@
+from bs4 import BeautifulSoup
+import logging
+
+
+def fulltext_indication(api_response_xml):
+    """Given XML from an API response, determine whether fulltext is available
+    for that item. Return True if fulltext is available, and False if it is
+    not."""
+    parseable_response_xml = BeautifulSoup(
+            api_response_xml,
+            features='xml')
+
+    doi_value = parseable_response_xml.find(
+            id='rft.doi').string
+
+    full_text_indicator_value = parseable_response_xml.find(
+            id='full_text_indicator')
+
+    logging.info(
+          'The value for the "full_text_indicator" key for doi "%(doi)s" is '
+          '"%(value)s".' % {
+                  'doi': doi_value,
+                  'value': full_text_indicator_value})
+
+    if(full_text_indicator_value is not None and
+       full_text_indicator_value.string == 'true'):
+        return 1
+    else:
+        return 0
+
+# =============================================================================
```

Comment:
This comment should be moved to the docstring, deleted, or encoded as a unit test.

Reply:
I've encoded these as unittests in 468e1e.